lib/_emerge/EbuildBuild.py

```diff
@@ -137,6 +137,13 @@ def _prefetch_exit(self, prefetcher):
 		pkg = self.pkg
 		settings = self.settings
 
+		quiet_setting = settings.get('PORTAGE_QUIET', False)
+		fetch_log = None
+		if quiet_setting:
+			fetch_log = os.path.join(
+					_emerge.emergelog._emerge_log_dir,
+					'emerge-fetch.log')
```

Comment:
This needs to be conditional on write access to `emerge-fetch.log` or a parent directory if the file doesn't exist yet, so we need some kind of conditional logic here. We can use the `portage.util.path.first_existing` function like this to check for write access:
```python
os.access(first_existing(fetch_log), os.W_OK)
```
It there's no write access to the log, then I suppose you'll have to allow the output to flow to directly stdout.

If there is write access, and output goes to the log, then we also need to ensure that error messages appropriately refer to the log file when fetch fails.

Reply:
I've implemented the permissions check as you requested. In terms of the error output pointing to the correct logfile, after digging into the code I can see that this behaviour is already implemented. `_proc_join_done()` in EbuildFetcher.py will raise an error and point to the correct logfile if we encounter an issue when prefetching.
I think this is ready for another review.

Reply:
Thanks! Merged in e26a08896f098ff93530efcd2d77d60d2eff0257.